image.go

```diff
@@ -12,10 +12,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Image within container
 type Image interface {
+	// Name of the image
```

Comment:
`Name returns the image name`?

Or something else, but need a verbs after `Name`